src/WinSparkleDotNet/WinSparkle.cs

```diff
@@ -0,0 +1,483 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace WinSparkleDotNet
+{
+    sealed public class WinSparkle
+    {
+        ////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Pardon my ignorance if I’m wrong, but these long comment separators don’t seem to be standard, are they? Unless this is standard practice, can you please remove them (and cut down on empty lines both inside the comments and between the `///` comment and the method)? This adds so much visual clutter that it’s hard to just read the file, my aim here is to make it more readable at a glance...